test/functional_test.js

```diff
@@ -417,7 +417,8 @@ function runOnPrefix(apiPrefix) {
         });
       });
 
-      it("should return a list of call-url with some information",
+      // Unable to test now that POST call-url is not available.
+      it.skip("should return a list of call-url with some information",
```

Comment:
We probably should update the functional tests to add the call-urls directly with a call to the database.